scipy/io/mmio.py

```diff
@@ -24,30 +27,29 @@
 #-------------------------------------------------------------------------------
 def mminfo(source):
     """
-    Queries the contents of the Matrix Market file 'filename' to
+    Queries the contents of the Matrix Market file 'source' to
     extract size and storage information.
 
     Parameters
     ----------
 
-    source : file
+    source : str or file-like
         Matrix Market filename (extension .mtx) or open file object
 
     Returns
     -------
 
     rows,cols : int
-       Number of matrix rows and columns
```

Comment:
rows, cols should be two separate entries, because it's ambiguous whether (rows, cols) is a tuple or two separate values.

Sorry, I realize you are only expanding, just hoping you'd consider fixing this while you're here.